### PR TITLE
Feature/adh 5112

### PR DESCRIPTION
**Pull request by workNpp**
https://tracker.yandex.ru/ADH-5112

### CODE REVIEW

File: smart-frontend/app/src/components/pages/ActionPage/ActionExecutionInfo/ActionExecutionInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useState } from 'react';
+import TabsBlock from '@uikit/Tabs/TabsBlock';
+import Tab from '@uikit/Tabs/Tab';
```
Comment (KuzminIA) [inline smart-frontend/app/src/components/pages/ActionPage/ActionExecutionInfo/ActionExecutionInfo.tsx]: is it possible to import `Tab` and `TabBlock` from `@uikit`

File: smart-frontend/app/src/components/pages/ActionPage/ActionHeader/ActionHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { type IconsNames, Title } from '@uikit';
+import Icon from '@uikit/Icon/Icon';
```
Comment (KuzminIA) [inline smart-frontend/app/src/components/pages/ActionPage/ActionHeader/ActionHeader.tsx]: please, import from `@uikit`

File: smart-frontend/app/src/components/pages/ActionPage/ActionPage.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useStore } from '@hooks';
+import ActionHeader from '@pages/ActionPage/ActionHeader/ActionHeader';
```
Comment (KuzminIA) [inline smart-frontend/app/src/components/pages/ActionPage/ActionPage.tsx]: you can use relative paths

File: smart-frontend/app/src/components/pages/ActionPage/useRequestAction.ts [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+  const debounceGetData = useDebounce(() => {
+    dispatch(getAction(parseInt(actionId)));
+  }, defaultDebounceDelay);
+
+  const debounceRefreshData = useDebounce(() => {
+    dispatch(refreshAction(parseInt(actionId)));
+  }, defaultDebounceDelay);
+
+  useRequestTimer(debounceGetData, debounceRefreshData, 1, isNeedUpdate, [actionId]);
```
Comment (KuzminIA) [inline smart-frontend/app/src/components/pages/ActionPage/useRequestAction.ts]: is it possible to move 1 to constants?

File: smart-frontend/app/src/components/uikit/Tooltip/AdvancedTooltips/BigTextTooltip/BigTextTooltip.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import s from './BigTextTooltip.module.scss';
+import type { TooltipProps } from '@uikit/Tooltip/Tooltip';
+import Tooltip from '../../Tooltip';
```
Comment (KuzminIA) [inline smart-frontend/app/src/components/uikit/Tooltip/AdvancedTooltips/BigTextTooltip/BigTextTooltip.tsx]: please, combine imports

File: smart-frontend/app/src/store/adh/action/actionSlice.ts [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+});
+
+const createInitialState = (): AdhActionSliceState => ({
+  action: null,
+  loadState: LoadState.NotLoaded,
+  isSomeError: false,
+});
+
+const actionSlice = createSlice({
+  name: 'adh/actions',
```
Comment (KuzminIA) [inline smart-frontend/app/src/store/adh/action/actionSlice.ts]: prefix must be `adh/action`

File: smart-frontend/app/src/components/pages/ActionPage/ActionExecutionInfo/ActionExecutionInfo.tsx (hunk 1 above)
Comment (workNpp) [reply smart-frontend/app/src/components/pages/ActionPage/ActionExecutionInfo/ActionExecutionInfo.tsx]: Yes, thnx!

File: smart-frontend/app/src/components/pages/ActionPage/ActionHeader/ActionHeader.tsx (hunk 2 above)
Comment (workNpp) [reply smart-frontend/app/src/components/pages/ActionPage/ActionHeader/ActionHeader.tsx]: Done, thnx!

File: smart-frontend/app/src/components/pages/ActionPage/ActionPage.tsx (hunk 3 above)
Comment (workNpp) [reply smart-frontend/app/src/components/pages/ActionPage/ActionPage.tsx]: You are right, thnx!

File: smart-frontend/app/src/components/pages/ActionPage/useRequestAction.ts (hunk 4 above)
Comment (workNpp) [reply smart-frontend/app/src/components/pages/ActionPage/useRequestAction.ts]: Moved!

File: smart-frontend/app/src/components/uikit/Tooltip/AdvancedTooltips/BigTextTooltip/BigTextTooltip.tsx (hunk 5 above)
Comment (workNpp) [reply smart-frontend/app/src/components/uikit/Tooltip/AdvancedTooltips/BigTextTooltip/BigTextTooltip.tsx]: Done!

File: smart-frontend/app/src/store/adh/action/actionSlice.ts (hunk 6 above)
Comment (workNpp) [reply smart-frontend/app/src/store/adh/action/actionSlice.ts]: Yes, thnx!

**Review by KuzminIA — APPROVED** [review]

**Review by remizov-arena — APPROVED** [review]

**Pull request merged**